Training/python/2018v1/Training_v0p1.py

```diff
@@ -491,8 +498,30 @@ def main(cfg: DictConfig) -> None:
         print("loss consts:",TauLosses.Le_sf, TauLosses.Lmu_sf, TauLosses.Ltau_sf, TauLosses.Ljet_sf)
 
         netConf_full = dataloader.get_net_config()
+        compile_loss = None if setup["loss"] is None else getattr(TauLosses,setup["loss"])
```

Comment:
- why do we need `compile_loss`? As far as I see it should always be `None`
- make sure that `loss` specified in `yaml` is used during the gradient descent.

Reply:
I forgot to wrap it inside DeepTauModel, let me fix it. The problem is that we want to try different losses but as it is hard-coded inside DeepTauModel that is not very elegant

Reply:
I think the best way is to pass the loss name to the `DeepTauModel` constructor.